src/utils.ts

```diff
@@ -1403,12 +1424,15 @@ export class newRect extends newShape {
     public size: Vertex = new Vertex(0, 0)
   ) {
     super();
-    this.path = this.buildPath();
+    this.buildPath();
```

Comment:
I think that `this.path = this.buildPath()` was actually better in order to avoid setting it in every overwritten method. Is there anything I am missing here ?

Reply:
It was a source of error when updating objects: if i did not explicitly write (for example for points) `this.points[index].path = this.points[index].buildPath()` instead of `this.points[index].buildPath()`, the points did not update.

It was not so practical. I find this way better